src/dune/dune.py

```diff
@@ -101,6 +103,10 @@ def create_node(self, nod):
         print("Creating node [" + nod.name() + "]")
         self._docker.execute_cmd(['mkdir', '-p', nod.data_dir()])
 
+    def get_current_nodeos_version(self):
+        stdout, stderr, exit_code = self._docker.execute_cmd(['nodeos', '--version'])
+        return stdout[1:].split('.')
```

Comment:
seems like either a regex or simply taking the whole string might future proof this a bit better.

Reply:
Good point.

Reply:
for example, what do we do with the trailing release info? e.g. `v4.1.0-dev`. Should patch really be `0-dev`?